addons/main.go

```diff
@@ -96,6 +98,18 @@ func main() {
 
 	ctx := ctrl.SetupSignalHandler()
 
+	webhooktls, err := webhook.NewTLSCredentials("self.signed.cert",
+		"tanzu.vmware",
+		"hostname",
+		[]string{"webhook-service", "webhook-service.default", "webhook-service.default.svc"},
+		time.Now(),
+		time.Hour*24*365)
+	if err != nil {
+		setupLog.Error(err, "Unable to generate tls certificates for webhooks")
+	} else {
+		webhooktls.WriteFiles()
+	}
+
```

Comment:
This whole section needs to be rework. 
We still need to get the list of webhooks which will need to be updated with the correct ca certificates.